### PR TITLE
Change dropdown menu to red

### PR DESCRIPTION
**Pull request by Ploypiti**
[Deploy link](https://ploypiti.github.io/2016.fossasia.org/)
(GCI task)

### CODE REVIEW

**Review by akshatnitd — APPROVED** [review]

File: css/theme-lava.css [hunk 1]
```diff
@@ -536,38 +535,71 @@ nav .container {
...
+    opacity: 1;
+    background-color: #E12B00;
+}
+
+
+
+.nav-dropdown{
+  display: none;
+  min-height:314px;
+
```
Comment (abishekvashok) [inline css/theme-lava.css]: Unwanted blank line

File: css/theme-lava.css [hunk 2]
```diff
@@ -618,6 +650,16 @@ nav .container {
 .sticky-nav .has-dropdown:before {
   color: #333333;
 }
+.sticky-nav .has-dropdown .nav-dropdown li a:hover
+{
```
Comment (abishekvashok) [inline css/theme-lava.css:645]: shift this to the previous line pls

File: css/theme-lava.css [hunk 3]
```diff
@@ -618,6 +650,16 @@ nav .container {
...
+  border-color: rgba(0,0,0,0);
+  color: white;
+}
+.sticky-nav .has-dropdown .nav-dropdown li:hover
+{
```
Comment (abishekvashok) [inline css/theme-lava.css]: shift this as well

**Review by abishekvashok — COMMENTED** [review]
These are just code lint errors, the code looks good if they are removed.

**Review by anshumanv — APPROVED** [review]
LGTM, please fix lint errors.

**Review by nzec — APPROVED** [review]
Awesome!

File: css/theme-lava.css (hunk 2 above)
Comment (realslimshanky) [reply css/theme-lava.css:645]: yes, a newline is required.

File: css/theme-lava.css [hunk 4]
```diff
@@ -800,7 +838,8 @@ nav .container {
   .nav-dropdown {
     position: relative;
     display: none;
-  }
+    min-height: 600px; 
+ }
```
Comment (realslimshanky) [inline css/theme-lava.css:834]: A newline is required after this.

File: css/theme-lava.css [hunk 5]
```diff
@@ -618,6 +649,13 @@ nav .container {
 .sticky-nav .has-dropdown:before {
   color: #333333;
 }
+.sticky-nav .has-dropdown .nav-dropdown li a:hover
+{
+  border-color: rgba(0,0,0,0);
+  color: white;
+  opacity: 1;
+  background-color: #E12B00;
+}
```
Comment (realslimshanky) [inline css/theme-lava.css:650]: A newline is required after this.

File: css/theme-lava.css [hunk 6]
```diff
@@ -536,38 +535,70 @@ nav .container {
...
+  display: block;
 }
+
+
 .has-dropdown:hover .nav-dropdown {
-  z-index: 10;
+  opacity: 1;
   max-height: 300px;
-  display: block;
 }
```
Comment (realslimshanky) [inline css/theme-lava.css:587]: A newline is required after this.

File: css/theme-lava.css [hunk 7]
```diff
@@ -536,38 +535,70 @@ nav .container {
...
+  padding-top: 9px;
+  padding-bottom: 9px;
+  padding-left: 2px;
   color: #333333;
+  -webkit-transition: all 0.2s;
 }
 .nav-dropdown li a:hover {
   border-color: rgba(0, 0, 0, 0);
+  color: white;
+}
```
Comment (realslimshanky) [inline css/theme-lava.css:572]: A newline is required after this.

File: css/theme-lava.css [hunk 8]
```diff
@@ -536,38 +535,70 @@ nav .container {
...
+.nav-dropdown li:hover {
+    opacity: 1;
+    background-color: #E12B00;
+}
+
+
```
Comment (realslimshanky) [inline css/theme-lava.css]: Please remove lines 580 and 581.

File: css/theme-lava.css [hunk 9]
```diff
@@ -536,38 +535,70 @@ nav .container {
...
-  margin-top: 12px;
+  margin-top: -1px;
 }
 .nav-dropdown li {
   opacity: 0;
   margin-right: 0px;
   float: none;
-  margin-bottom: 18px;
+  -webkit-transition: all 0.3s;
 }
```
Comment (realslimshanky) [inline css/theme-lava.css:560]: A newline is required after this.

File: css/theme-lava.css [hunk 10]
```diff
@@ -536,38 +535,70 @@ nav .container {
...
```
Comment (realslimshanky) [inline css/theme-lava.css:535]: A newline is required after this.

File: css/theme-lava.css [hunk 11]
```diff
@@ -92,7 +92,6 @@ img {
 .footer-container,
 .nav-container,
 nav {
-  max-width: 1600px;
   margin: 0px auto;
 }
```
Comment (realslimshanky) [inline css/theme-lava.css:96]: A newline is required after this.

**Review by realslimshanky — CHANGES_REQUESTED** [review]
To make the css syntax even everywhere a newline is required after every style declaration. Either follow the reviews or remove newline after each declaration. The goal is to make syntax even across the file.

**Comment by Ploypiti** [comment]
Removed exceeded lines, squashed. Please merge if possible.

**Review by realslimshanky — APPROVED** [review]
LGTM

**Review by sayak119 — APPROVED** [review]

**Comment by anshumanv** [comment]
Everything seems to be in place.

**Pull request merged**